com.playeveryware.eos/Runtime/Core/Common/SetOfNamed.cs

```diff
@@ -103,31 +148,7 @@ private string GetNewItemName()
         /// </returns>
         public bool Add()
         {
-            return Add(default);
-        }
-
-        /// <summary>
-        /// Adds a new item with the indicated name to the SortedSetOfNamed
-        /// items.
-        /// </summary>
-        /// <param name="name">
-        /// The name of the item to add to the collection.
-        /// </param>
-        /// <param name="value">
-        /// The value to add to the collection.
-        /// </param>
-        /// <returns>
-        /// True if the add was successful, false otherwise.
-        /// </returns>
-        public bool Add(string name, T value)
-        {
-            // Add to the collection using the base implementation, so long as
-            // the name for the new item is not used for another named item 
-            // already in the collection, and so long as the value is not the
-            // same as the value of another item in the collection.
-            return !ContainsName(name) &&
-                   !Contains(value) &&
-                   base.Add(Named<T>.FromValue(value, name));
+            return Add(default(T));
```

Comment:
Should this use logic similar to the above with `value ??= typeof(T).IsValueType ? default : new T();`?